03-single-hop-ipv6-icmp/03-single-hop-ipv6-icmp.md

```diff
@@ -48,6 +48,7 @@ ICMPv6 echo request/reply exchange between two native nodes.
 Task #04 - ICMPv6 stress test on native (1 hour)
 ================================================
 ### Description
+This tests will require `CFLAGS=-DGNRC_IPV6_NIB_NUMOF=10` to run.
```

Comment:
No. This is completely against the point of this test. The node should be able to cache out the nodes appropriately enough.